src/saving/SaveManagerGUI.cs

```diff
@@ -177,6 +201,22 @@ private void OnConfirmDeleteSelected()
         RefreshList();
     }
 
+    private void OnConfirmDeleteOld()
+    {
+        string autoSaveNames = string.Join(", ", SaveHelper.CleanUpOldSavesOfType("auto_save"));
```

Comment:
I think you could use just a single string, or maybe a stringbuilder if a string creation is too slow.
Very rough code:
```
string message = string.Join(", ", SaveHelper.CleanUpOldSavesOfType("auto_save"));
if(message.Length > 0)
    message += ", ";
message += string.Join(", ", SaveHelper.CleanUpOldSavesOfType("quick_save"));
```